lava/assetlist.json

```diff
@@ -0,0 +1,32 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "lava-testnet-1",
```

Comment:
please change to just 'lava'

Reply:
Corrected:
[dae51fa](https://github.com/cosmos/chain-registry/pull/2244/commits/dae51fa9ed2a0b9e92a8e6dbd1a6de701cc726e1)